packages/domain/projet/src/lauréat/lauréat.aggregate.ts

```diff
@@ -228,6 +232,38 @@ export class LauréatAggregate extends AbstractAggregate<LauréatEvent> {
     }
   }
 
+  vérifierQueLeLauréatEstSansAbandon() {
+    if (this.projet.statut.estAbandonné()) {
+      throw new ProjetAbandonnéError();
+    }
+
+    if (this.abandon.statut.estEnCours()) {
+      throw new ProjetAvecDemandeAbandonEnCoursError();
+    }
+  }
+
+  vérifierQueLeLauréatEstSansAchèvement() {
```

Comment:
```suggestion
  vérifierQueLeLauréatEstPasAchevé() {
```
est tout de même plus clair et plus lisible